### PR TITLE
Added initial set up for fluent-bit.

### PR DESCRIPTION
**Pull request by DavidKorczynski**
Added an initial set up for fluent-bit. Waiting for maintainers to accept and give an email (https://github.com/fluent/fluent-bit/pull/2090), then I will update here.

### CODE REVIEW

File: projects/fluent-bit/build.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+cd fluent-bit/build
+cmake -DFLB_RECORD_ACCESSOR=Off -DFLB_STREAM_PROCESSOR=Off -DFLB_LUAJIT=OFF ..
+make || true
```
Comment (Dor1s) [inline projects/fluent-bit/build.sh:19]: why should we ignore failing `make`?

Comment (Dor1s) [reply projects/fluent-bit/build.sh:19]: ping on this question. Good to go otherwise

Comment (DavidKorczynski) [reply projects/fluent-bit/build.sh:19]: The only reason is that some of the plugins (not used by the fuzzer) fails with the OSS-Fuzz flags. I was inspired by using the || true from the binutils project here at OSS-Fuzz. 
I will keep working with the maintainers on integrating it further into the project so we can remove this in the future.

Comment (Dor1s) [reply projects/fluent-bit/build.sh:19]: Thanks, let's land this for now, but ideally there should be some make option to disable that failing plugin, if it's not needed for building fuzz targets.

**Pull request merged**